src/main/java/org/newrelic/nrjmx/Arguments.java

```diff
@@ -64,9 +65,13 @@ static Options options() {
             Option help = Option.builder("h")
                 .longOpt("help").desc("Show help").hasArg(false).build();
             options.addOption(help);
+
             Option remote = Option.builder("r")
-                .longOpt("remote").desc("Remote JMX mode").hasArg(false).build();
+                    .longOpt("remote").desc("Remote JMX mode").hasArg(false).build();
             options.addOption(remote);
+            Option remoteJBossStandalone = Option.builder("s")
```

Comment:
we should put string literals as constants, to avoid mistakes on modifying `remoteJBossStandalone` here and not in the `Option` construction.